CMakeLists.txt

```diff
@@ -453,10 +459,15 @@ if(QUIC_TLS STREQUAL "openssl")
                 OUTPUT ${QUIC_BUILD_DIR}/submodules/openssl/release/makefile
                 COMMAND perl ${CMAKE_CURRENT_SOURCE_DIR}/submodules/openssl/Configure ${OPENSSL_CONFIG_FLAGS} --prefix=${OPENSSL_DIR}/release)
 
-            add_custom_target(OpenSSL_Build
+            add_custom_command(
+                OUTPUT ${LIBSSL_PATH} ${LIBSSL_DEBUG_PATH} ${LIBCRYPTO_PATH} ${LIBCRYPTO_DEBUG_PATH}
```

Comment:
We need to make sure this doesn't break MSVC. I think the reason I didn't have these is because of the generator expressions they don't get built in each mode.